modules/nw-dual-stack-convert.adoc

```diff
@@ -0,0 +1,76 @@
+[id="nw-dual-stack-convert_{context}"]
+= Converting to a dual network stack cluster network
+
+As a cluster administrator, you can convert your single stack cluster network to a dual-stack cluster network.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* Your cluster is using the OVN-Kubernetes cluster network provider.
+
+.Procedure
+
+. To specify IPv6 address blocks for the cluster and service networks, create a file containing the following YAML:
```

Comment:
This is an implementation detail, an example on how to do it, it can be done in different ways
I would use something along these lines, you'll find better wording for sure
"To add a new IPv6 address block for the cluster and service network, you have to modify current Openshift network configuration. An example on how to do it, is creating a file containing a patch like in the following yaml:"

Reply:
@aojea, that makes sense. In the interests of expediency, I'd like to retain the existing procedure. For docs we mostly try to provide just one way to accomplish a task.

Reply:
Ahh ok, didn't know it, then disregard the comment